### PR TITLE
Mobile responsive and removing style.css pre-loading

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
Removed, pre-loading `style.css` file. It's ending up loading multiple times in su-paths. 
- One from the response header.
- Fromt the `html` entry `./style.css`

Screenshot 2022-01-18 at 5 03 38 PM
Screenshot 2022-01-18 at 5 03 32 PM
Screenshot 2022-01-18 at 5 03 25 PM

### CODE REVIEW

**Comment by JayaKrishnaNamburu** [comment]
And one more thing, do we need `Exports` for displaying all files. I personally feel, it's not adding much value. Apart from confusion, since these files can't be loaded and used directly anyways. I suggest, we remove it from the UI. What you say ?

File: src/header.jsx [hunk 1]
```diff
@@ -5,26 +5,34 @@ import { Nav } from "./nav.js";
 
 function Header() {
   return (
-    <jspm-package-header>
-      <header>
-        <Logo />
-        <Search />
+    <div>
```
Comment (fusionstrings) [inline src/header.jsx:8]: Use of custom element syntax is intentional here.

Comment (JayaKrishnaNamburu) [reply src/header.jsx:8]: Yes, but that's making hard to write media-queries. They don't inherit fields. And instead overwriting them 😅

Comment (JayaKrishnaNamburu) [reply src/header.jsx:8]: If there is any specific usage for this, i will wrap this with those tags

**Comment by fusionstrings** [comment]
Thanks for PR and being hawk-eye 🥇

**Comment by fusionstrings** [comment]
I guess I am gonna merge it.

**Pull request merged**